src/TransitionSystems/location_id.rs

```diff
@@ -21,13 +23,52 @@ impl LocationID {
             "consistency: {}",
             string
         ))
+        .unwrap()
         .remove(0);
 
         match query {
             QueryExpression::Consistency(x) => (*x).into(),
             _ => unreachable!(),
         }
     }
+
+    /// This function is used when you want to compare [`LocationID::Simple`] location to [`LocationID::AnyLocation`].
+    /// [`LocationID::AnyLocation`] should always be true when compared to [`LocationID::Simple`]
+    /// ```
+    /// use reveaal::TransitionSystems::LocationID;
+    /// // Make two locations where `a` has LocationID::AnyLocation
+    /// let a = LocationID::Quotient(Box::new(LocationID::Simple("L5".to_string())),
+    ///                              Box::new(LocationID::AnyLocation()));
+    ///
+    /// let b = LocationID::Quotient(Box::new(LocationID::Simple("L5".to_string())),
+    ///                              Box::new(LocationID::Simple("L1".to_string())));
+    ///
+    /// assert!(a.cmp_partial_locations(&b));
+    /// ```
+    pub fn cmp_partial_locations(&self, other: &LocationID) -> bool {
```

Comment:
i dont like the cmp shorthand, doesnt feel rust like. Maybe just "compare" instead

Reply:
fixed

Reply:
e9